CS342MemberViews/Sources/AbhinavAgarwal/LinkedInUsername.swift

```diff
@@ -0,0 +1,40 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct LinkedInUsername: View {
```

Comment:
One of the tasks of the Assignment 1 is a creative and nice addition beyond the views that we have in the application.
Please try to add a creative and unique solution to your PR.